tests/acceptance/api_test.go

```diff
@@ -1870,7 +1870,7 @@ func TestApi(t *testing.T) {
 		t.Run("When there is a status parameter", func(t *testing.T) {
 			resp := apiRequest(t, &apiRequestInput{
 				method: "GET",
-				url:    apiURL + "/accounts?status=" + string(db.NotReady),
+				url:    apiURL + "/accounts?accountStatus=" + string(db.NotReady),
```

Comment:
Match the json payload verbage for account status

Reply:
This is a contract (breaking) change. Definitely needs to be called out in CHANGELOG and really should be avoided altogether now that this is public. Almost better to rename the variable....

Reply:
This would be possible to keep it the same.  Its a bummer because the JSON response is accountStatus, the dynamodb is AccountStatus and the query is status.   Also in all fairness the query of account status hasn't been released yet.

https://github.com/Optum/dce/blob/master/pkg/api/response/account.go#L22
https://github.com/Optum/dce/blob/master/pkg/db/model.go#L11

I don't mind calling out a breaking change but it would have been nice to make this accurate instead of perpetuating some of the data silliness we have now.